src/request.js

```diff
@@ -98,6 +98,9 @@ export default class Request {
 			input.compress : true;
 		this.counter = init.counter || input.counter || 0;
 		this.agent = init.agent || input.agent;
+		this.lookup = init.lookup || input.lookup;
+		this.family = init.family || input.family;
+		this.localAddress = init.localAddress || input.localAddress;
```

Comment:
These should probably have a default value as well, like

```js
this.lookup = init.lookup || input.lookup || dns.lookup;
this.family = init.family || input.family || null;
this.localAddress = init.localAddress || input.localAddress || null;
```

Reply:
Done in 699fc3268f6cf60069d40f4fcad3ec4f812c685e, although I don't really agree too strongly here; we did not do this for `agent` either, and by passing in `dns.lookup` as the default we replicate the default value of a dependency, which might change in the future. Leaving it undefined when not specified would take a more neutral stance, imho. But it's not that important really, so I'm fine with the change if You are.